packages/amd-loader/amd-loader.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+(function() {
+
+function Registry() {
```

Comment:
Even though what's inside the module closure is top-level, I'd still indent it. Initially I thought some things below were outside of the IIFE for some reason.